apps/main/src/dex/components/PagePool/Swap/index.tsx

```diff
@@ -88,15 +86,32 @@ const Swap = ({
 
   const poolId = poolData?.pool?.id
   const haveSigner = !!signerAddress
+
   const userFromBalance = userPoolBalances?.[formValues.fromAddress]
   const userToBalance = userPoolBalances?.[formValues.toAddress]
+
   const fromUsdRate = usdRatesMapper[formValues.fromAddress]
   const toUsdRate = usdRatesMapper[formValues.toAddress]
 
-  const { selectList, swapTokensMapper } = useMemo(
-    () => getSwapTokens(tokensMapper, poolDataCacheOrApi),
-    [poolDataCacheOrApi, tokensMapper],
-  )
+  const { selectList, swapTokensMapper } = useMemo(() => {
+    const { selectList, swapTokensMapper } = getSwapTokens(tokensMapper, poolDataCacheOrApi)
+
+    return {
+      selectList: selectList
+        .filter((token) => !!token)
+        .map((token) => ({
+          chain: network?.networkId ?? '',
+          address: token?.address as `0x${string}`,
+          symbol: token?.symbol,
+          label: '',
```

Comment:
Why is this empty? And why do we use an empty string instead of undefined?
```suggestion
```

Reply:
I prefer explicitness and want to avoid null & undefined. Will make this one optional though